tasks/nginx-stateless.yml

```diff
@@ -5,6 +5,10 @@
 
 - set_fact:
     thumbprint: "{{ cmdoutput.stdout.split()[0] }}"
+- name: "Create nginx folder"
+  file:
+    path: "/etc/nginx"
+    state: "directory"
```

Comment:
Do we need this?

Reply:
Yes, if acmetool is installed before nginx, the /etc/nginx/ folder is not already created, so the /etc/nginx/acmetool-location.conf can't be put in place.

Creating it here, and using this role before nginx, allows to run a full https deployment on one pass.